partials/time_to_retirement.html

```diff
@@ -15,111 +15,121 @@
             </div>
         </div>
         <div class="row cannot-retire col-xs-12 col-sm-8 col-sm-offset-2" ng-show="retirement.never_retire_shown">
+        	<div class="big-number-bg">
+                <div class="big-number-wrapper">
+                	<i class="fa fa-remove icon-answer"></i>
+                </div>
+         	</div>
             <div class="col-xs-12 no-retire-message">With your current cash flow trajectory, your future passive income will never safely exceed your expenses. <a href="#/income">Try calculating your countdown with a lower expense rate or higher income.</a>
             </div>
         </div>
         <div class="row cannot-retire col-xs-12 col-sm-8 col-sm-offset-2" ng-show="retirement.already_retired_shown">
-            <div class="col-xs-12 no-retire-message">You're already financially independent.</div>
+        	<div class="big-number-bg">
+                <div class="big-number-wrapper">
+                	<i class="fa fa-check icon-answer"></i>
+                </div>
+         	</div>
+            <div class="until-fi">You're already financially independent.</div>
         </div>
     </div>
     <div class="container-fluid">
         <div class="row">
             <div class="graph-section supporting-content">
                 <div class="supporting-content-wrapper">
-                    <div class="col-md-6" id="graph-wrapper">
+                    <div class="col-md-8" id="graph-wrapper">
                         <div id="retirement-graph"></div>
+                        <div id="explain-text">Financial independence is based on <a href="#/about">the 4% Safe Withdrawal Rate</a></div>
                     </div>
-                    <div class="supporting-section-body col-md-6">
+                    <div class="supporting-section-body col-md-4">
                         <div id="expenses-control" class="input-control col-md-offset-3 col-md-5">
-                            <div class="input-group">
-                                <span class="input-group-btn">
-							        <button class="btn btn-default glyphicon-minus" type="button" ng-click="expenses = expenses - 100"></button>
-							    </span>
-                                <input ng-currency type="text" class="form-control expenses-field" ng-model="expenses" />
-                                <span class="input-group-btn">
-							        <button class="btn btn-default glyphicon-plus" type="button" ng-click="expenses = expenses + 100"></button>
-							    </span>
+                            <div class="">
+                            	<div class="">
+						        	<button class="btn btn-default glyphicon-plus control-increase-button control-change-button" type="button" ng-click="expenses = expenses + 100"></button>

```

Comment:
Yeah I agree. We need to make each of these controls into directives. This is on my tech debt roadmap. BUT, the question is: now or later? My instinct is that this is a task we can leave till post-launch, and live with the non-ideal architecture for now. I guess I'd rather spend time on functionality I think our v1 needs, so we can start learning from users, and optimize later.

BUT, I differ to you on tech stuff like this. If you think I should refactor into directives so logic is contained in the main module, I can tackle that, no problem. It's just time.